src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -169,22 +174,28 @@ protected override Task CreateVersionsAsync(CancellationToken cancellationToken)
                 && (latestStableVersion.version == null || latestPrerelease.version > latestStableVersion.version) &&
                 !latestPrerelease.version.Equals(installedVersion))
             {
-                _versions.Add(new DisplayVersion(latestPrerelease.version, Resources.Version_LatestPrerelease, isDeprecated: latestPrerelease.isDeprecated));
+                VersionRange latestPrereleaseVersionRange = VersionRange.Parse(latestPrerelease.version.ToString(), false);
+                _versions.Add(new DisplayVersion(latestPrereleaseVersionRange, latestPrerelease.version, Resources.Version_LatestPrerelease, isDeprecated: latestPrerelease.isDeprecated));
             }
 
             // Add latest stable if needed
             if (latestStableVersion.version != null &&
                 !latestStableVersion.version.Equals(installedVersion))
             {
-                _versions.Add(new DisplayVersion(latestStableVersion.version, Resources.Version_LatestStable, isDeprecated: latestStableVersion.isDeprecated));
+                VersionRange latestStableVersionRange = VersionRange.Parse(latestStableVersion.version.ToString(), false);
+                _versions.Add(new DisplayVersion(latestStableVersionRange, latestStableVersion.version, Resources.Version_LatestStable, isDeprecated: latestStableVersion.isDeprecated));
             }
 
+            // Only the current installed version is displayed, we update the separator so its not removed from the list when filtering.
+            IsBeforeNullSeparator = _versions.Count == 1;
+
             // add a separator
-            if (_versions.Count > 1)
+            if (_versions.Count > 0)
```

Comment:
What was the bug here? 
Now we will have a separator even for 1 item? Can you screenshot how this looks.

Reply:
No bug here, I think I changed this to try something and forgot to revert